package.json

```diff
@@ -231,7 +231,7 @@
     "test": "yarn test:types && yarn test:server && yarn test:app",
     "test:app": "jest --forceExit",
     "test:server": "cd server && yarn test",
-    "test:types": "tsc --project tsconfig.build.json --noEmit && cd server && tsc --noEmit",
+    "test:types": "tsc --project tsconfig.test.json --noEmit && cd server && tsc --noEmit",
```

Comment:
why do we need yet another config?

Reply:
I use a different config for tests very often (like in most of my projects) because I don't want to type everything strictly while writing unit tests. So I disable noImplicitAny, as an example, just for tests. I think it is a good compromise of getting the best out of typescript for testing purposes without wasting too much time.

Reply:
removed it since we already have the build config